vividus-plugin-applitools/src/main/resources/ufg-applitools-visual-comparison.ftl

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Visual tests result table</title>
+    <link rel="stylesheet" href="../../styles.css"/>
+    <link rel="stylesheet" href="../../webjars/bootstrap/3.4.1/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="../../webjars/bootstrap-toggle/2.2.2/css/bootstrap-toggle.min.css">
+</head>
+<body>
+    <style>
+    body {
+        background-color: #EEEEEE;
+    }
+    </style>
+    <div class="container-fluid">
+        <h3>Baseline name: ${result.baselineName}</h3>
+        <table class="table table-hover">
+            <thead>
+            <tr>
+                <th scope="col">Status</th>
+                <th scope="col">Test</th>
+                <th scope="col">OS</th>
+                <th scope="col">Browser</th>
+                <th scope="col">Viewport</th>
+                <th scope="col">Link</th>
+            </tr>
+            </thead>
+            <tbody>
+                <#list result.testResults as testResult>
+                    <#if testResult.isPassed()>
+                        <tr class="success">
+                    <#else>
+                        <tr class="danger">
+                    </#if>
```

Comment:
we use own styles to mark passed/failed results

Reply:
moved styles to webjars/vividus/style.css in allure-adaptor, did the same with background for body, don't think it will break anything